components/projects/project-form.tsx

```diff
@@ -0,0 +1,296 @@
+          <div className="w-full space-y-3">
+            <Label htmlFor="name" className="text-xs">
+              NAME
+            </Label>
+            <Input
+              {...register("name")}
+              id={"name"}
+              aria-invalid={Boolean(errors.name)}
+              className="mt-1 w-full"
+              onChange={(e) => {
+                setValue("name", e.target.value)
+                if (action === "new") {
+                  const slug = createSlug(e.target.value)
+                  setValue("slug", slug)
+                }
+              }}
+            />
+            {errors.name && (
+              <p className="pt-1 text-xs text-error-solid" role="alert">
+                {errors.name?.message}
+              </p>
+            )}
+          </div>
+
+          <div className="w-full space-y-3">
+            <Label htmlFor="slug" className="text-xs">
+              SLUG
+            </Label>
+            <Input
+              readOnly
+              {...register("slug")}
+              id={"slug"}
+              aria-invalid={Boolean(errors.slug)}
+              className="mt-1 w-full"
+            />
+            {errors.slug && (
+              <p className="pt-1 text-xs text-error-solid" role="alert">
+                {errors.slug?.message}
+              </p>
+            )}
+          </div>
+        </div>
+
+        <div className="flex flex-col space-y-6 md:flex-row md:space-x-4 md:space-y-0">
+          <div className="w-full space-y-3">
+            <Label htmlFor="custom_domain" className="text-xs">
+              CUSTOM DOMAIN
+            </Label>
+            <Input
+              {...register("custom_domain")}
+              id={"custom_domain"}
+              aria-invalid={Boolean(errors.name)}
+              className="mt-1 w-full"
+              onChange={(e) => {
+                setValue("custom_domain", e.target.value)
+              }}
+            />
+            {errors.name && (
+              <p className="pt-1 text-xs text-error-solid" role="alert">
+                {errors.custom_domain?.message}
+              </p>
+            )}
+          </div>
+
+          <div className="w-full space-y-3">
+            <Label htmlFor="subdomain" className="text-xs">
+              SUBDOMAIN
+            </Label>
+            <Input
+              {...register("subdomain")}
+              id={"subdomain"}
+              aria-invalid={Boolean(errors.name)}
+              className="mt-1 w-full"
+              onChange={(e) => {
+                setValue("subdomain", e.target.value)
+              }}
+            />
+            {errors.slug && (
+              <p className="pt-1 text-xs text-error-solid" role="alert">
+                {errors.subdomain?.message}
+              </p>
+            )}
+          </div>
+        </div>
+
+        <div className="space-y-3">
+          <Label htmlFor="logo" className="text-xs">
+            LOGO (optional)
```

Comment:
I just updated the ui for using the new form component. I suggest to update this part with that.